jhub_apps/templates/style.css

```diff
@@ -383,6 +383,10 @@ a:focus {
     border: unset;
 }
 
+.card-dialog-body-wrapper {
+    padding: 0 24px 20px !important;
+}
+
```

Comment:
Fix spacing issue in modal dialogs, which are currently being overridden by JupyterLab styling